staff/ruben-ponce/precourse/4-skylab-airlines-pro-master/index.js

```diff
@@ -0,0 +1,196 @@
+console.log("El precio medio de nuestros vuelos es de: " + (costTotal/numeroVuelos)+"€.");
+
+//Vuelos que efectuan escalas
+console.log("Hoy hay " + escalas + " vuelos que efectúan escalas.");
+
+//Últimos vuelos del día
+var destino = "";
+for (var i = flights.length-5; i < flights.length; i++) {
+  destino += flights[i].to + ", ";
+};
+
+//Modificando el string
+destino1 = destino.slice([0], destino.length-2) + ".";
+var quit = destino1.lastIndexOf(",");
+destino2 = destino1.slice([0],quit) + " y" + destino1.slice(quit+1);
+
+console.log("Nuestros últimos 5 vuelos tienen como destino: " + destino2);
+
+//PRO - validación de usuario
+var usuario = prompt("Introduzca si es usted ADMIN o USER: ", "user").toUpperCase();
+
+//USER
+if (usuario === "USER") {
+
+  var masAlto = param => {
+    console.log("Los vuelos con mayor precio a " + param + "€ son:");
+    for (let index in flights) {
+      if (flights[index].cost > param) {
+        console.log(flights[index])
+      };
+    };
+  }
+
+  var masBajo = param => {
+    console.log("Los vuelos con menor precio a " + param + "€ son:")
+    for (let index in flights) {
+      if (flights[index].cost < param) {
+        console.log(flights[index])
+      };   
+    };
+  }
+
+  var igualQ = param => {
+    console.log("Los vuelos con precio igual a " + param + "€ son:")
+    for (let index in flights) {
+      if (flights[index].cost == param) {
+        console.log(flights[index])
+      };
+    };
+  }
+
+  var comprar = param => {
+    if (flights[param].id) {
+      console.log("Vuelo comprado:");
+      console.log(flights[param]);
+      console.log("Gracias por su compra, vuelva pronto.");
+    } else return console.log("El id del vuelo no existe.");
+  }
+
+  var preguntarAction = () => {
+    var action = prompt('Como usuario puedes:\n -Filtrar por precio, introduce: "mayor", "menor" o "igual".\n -Comprar un vuelo introduce: "comprar".\n -Para terminar introduce: "salir". ').toLowerCase();
+    
+    if (action === "mayor" || action === "menor" || action === "igual") {
+      var parame = prompt("Introduce el precio a filtrar");
+    } else if (action === "comprar") {
+      var parame = prompt("Introduce el ID del vuelo");
+    };
+    
+    switch(action) {
+      case "mayor":
+        masAlto(parame);
+        return preguntarAction();
+      case "menor":
+        masBajo(parame);
+        return preguntarAction();
+      case "igual":
+        igualQ(parame);
+        return preguntarAction();
+      case "comprar":
+        comprar(parame);
+        return preguntarAction();
+      case "salir":
+        return console.log("Hasta la próxima!");
+      default:
+        alert("Error en la instrucción");
+        return preguntarAction();
+    };
+  }
+  preguntarAction();
+};
+
+//ADMIM
+if (usuario === "ADMIN") {
+  
+//Mostrar vuelos   
+  var ver = () => {
+    for (let index in flights) {
+      console.log(flights[index]);
+    };
+  }
+
+//Crear vuelos
+  var crear = () => {
```

Comment:
Here you could just change the name of the function to crearVuelos() and then just remove the comment.